exporter/awsemfexporter/metric_translator.go

```diff
@@ -139,6 +171,9 @@ func (mt metricTranslator) translateOTelToGroupedMetric(rm pmetric.ResourceMetri
 		}
 	}
 
+	entity, resourceAttributes := fetchEntityFields(rm.Resource().Attributes())
+	resourceAttributes.CopyTo(rm.Resource().Attributes())
```

Comment:
Are we removing the additional entity attributes from resource attributes in this step?

Reply:
Have we verified the EMF logs if these entity attributes are removed from the logs published to cloudwatch?

Reply:
> Are we removing the additional entity attributes from resource attributes in this step?

added a unit test that confirms this behavior